src/components/pages/HomePage/index.jsx

```diff
@@ -0,0 +1,235 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+// DOM bindings for React Router
+import { Redirect } from "react-router-dom";
+//> Additional
+// Typing animations
+import Typed from "react-typed";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBContainer,
+  MDBSmoothScroll,
+  MDBRow,
+  MDBCol,
+  MDBCard,
+  MDBBtn,
+  MDBIcon,
+} from "mdbreact";
+
+//> Components
+import { UserActionCard } from "../../molecules";
+//> CSS
+import "./homepage.scss";
+//> Images
+import imageRanking from "../../../assets/body/ranking.png";
+import imageProfiles from "../../../assets/body/profiles.png";
+import imageTrophy from "../../../assets/body/trophy.png";
+//#endregion
+
+//#region > Components
+class HomePage extends React.Component {
```

Comment:
Add description: `This component displays the landing page including login and register.`